src/main/java/racing/domain/Racing.java

```diff
@@ -0,0 +1,34 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import racing.RandomGenerator;
+
+public class Racing {
+
+  List<Car> cars = new ArrayList<>();
```

Comment:
```suggestion
  private final List<Car> cars = new ArrayList<>();
```
외부에 노출되지 않는 정보에 대해서는 객체의 안정성을 위해 불변을 유지하면 좋을 것 같아요 😄